source/sessions/driver-sessions.rst

```diff
@@ -908,17 +935,13 @@ as follows:
 
 .. code:: typescript
 
-    {
-        ...
-        $clusterTime : {
-            clusterTime : <BsonTimestamp>,
-            signature : {
-                hash : <BsonBinaryData>,
-                keyId : <BsonInt64>
-            }
-        },
-        ...
-    }
+  interface ClusterTime {
+    clusterTime: Timestamp;
+    signature: {
+      hash: Binary;
+      keyId: Int64;
+    };
+  }
```

Comment:
I prefer the previous code which shows the whole response including the "$clusterTime" field.

Reply:
reformatted (in the other PR) lmk if it captures it well, the `ClusterTime` interface is referenced in other message examples but yea it's a good reminder to bring up here that it's under a dollar prefix property.